gems/aws-sdk-sns/spec/message_verifier_spec.rb

```diff
@@ -80,15 +98,19 @@ module SNS
       describe '#authenticate!' do
 
         it 'returns true for a valid message' do
-          expect(verifier.authenticate!(message)).to be(true)
+          expect(verifier.authenticate!(message_SHA1)).to be(true)
+        end
+
+        it 'returns true for a valid message' do
+          expect(verifier.authenticate!(message_SHA256)).to be(true)
```

Comment:
Do we want or need any more tests around message_SHA256? Are other things like raising when SigningCertURL is not valid also happening with a SHA256 message?